i18n/en.pot

```diff
@@ -5,8 +5,8 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2025-02-04T09:15:52.687Z\n"
-"PO-Revision-Date: 2025-02-04T09:15:52.687Z\n"
+"POT-Creation-Date: 2025-02-19T09:55:28.790Z\n"
+"PO-Revision-Date: 2025-02-19T09:55:28.790Z\n"
```

Comment:
No need to commit. Not sure why, mdsync seems to be the only project that updates the timestamp when no changes actually exist.